src/WebJobs.Script/Utility.cs

```diff
@@ -278,8 +278,12 @@ public static bool IsJson(string input)
             }
 
             input = input.Trim();
-            return (input.StartsWith("{", StringComparison.OrdinalIgnoreCase) && input.EndsWith("}", StringComparison.OrdinalIgnoreCase))
-                || (input.StartsWith("[", StringComparison.OrdinalIgnoreCase) && input.EndsWith("]", StringComparison.OrdinalIgnoreCase));
+            return IsJsonObject(input) || (input.StartsWith("[", StringComparison.OrdinalIgnoreCase) && input.EndsWith("]", StringComparison.OrdinalIgnoreCase));
+        }
+
+        public static bool IsJsonObject(string input)
+        {
+            return input.Trim().StartsWith("{", StringComparison.OrdinalIgnoreCase) && input.EndsWith("}", StringComparison.OrdinalIgnoreCase);
```

Comment:
is this enough to check for a Json Object?  Check if parsing the string succeeds: JToken.Parse(input)?

Reply:
Used JToken.Parse - good idea :)